onebusaway-android/src/main/java/org/onebusaway/android/report/ui/ReportProblemFragmentBase.java

```diff
@@ -105,16 +88,22 @@ public void onActivityCreated(Bundle savedInstanceState) {
 
     @Override
     public View onCreateView(LayoutInflater inflater,
-            ViewGroup root, Bundle savedInstanceState) {
+                             ViewGroup root, Bundle savedInstanceState) {
         if (root == null) {
             // Currently in a layout without a container, so no
             // reason to create our view.
             return null;
         }
+
         return inflater.inflate(getLayoutId(), null);
     }
 
     @Override
+    public void onResume() {
```

Comment:
You should be able to remove `onResume()` here, as it's just making a call to `super`.